docs/src/modules/utils/useRouterExtra.ts

```diff
@@ -2,8 +2,9 @@ import { useRouter } from 'next/router';
 
 export default function useRouterExtra() {
   const router = useRouter();
-  const asPathWithoutLang = router.asPath.replace(/^\/[a-zA-Z]{2}\//, '/');
-  let product = asPathWithoutLang.replace(/^\/([^/]+)\/.*/, '$1');
+
+  const asPathWithoutLang = router.asPath.replace(/^\/+[a-zA-Z]{2}\//, '/');
+  let product = asPathWithoutLang.replace(/^\/+([^/]+)\/.*/, '$1');
```

Comment:
Found this edge case when the URL has double `//`, e.g. clicking on the link in GoogleAnalytics does not toggle the active page.

<img width="572" alt="image" src="https://user-images.githubusercontent.com/18292247/217870697-4211492d-ef30-4dd6-80c8-78f0acbc7f86.png">

Reply:
Oh lol, Netlify, what did you do https://mui.com////////joy-ui/getting-started/templates/ is a 200 😂

> Google Analytics

I wonder if we have a misconfiguration in Google Analytics. Where do you find this link?

Reply:
Normal pageviews table
https://analytics.google.com/analytics/web/?pli=1#/my-reports/OpeHYRZ_SHGdsb0jfY5JNQ/a106598593w159022482p160376982/_u.date00=20221101&_u.date01=20221130&_.gSectionId=626/

<img width="881" alt="image" src="https://user-images.githubusercontent.com/18292247/218016504-da2e80dd-fd45-427f-9d27-7e885130276b.png">

Reply:
Oh, ok. Fixed, I followed https://webapps.stackexchange.com/questions/35341/google-analytics-adds-additional-slash-after-domain-when-opening-a-page. We had a leading slash in:

<img width="392" alt="Screenshot 2023-02-10 at 13 06 34" src="https://user-images.githubusercontent.com/3165635/218089050-b342c6bb-ac55-46e5-9e0f-2e6ff788c1a8.png">

Reply:
Ah, thanks!